src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,63 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed is-right">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={classNames(
+                  { 'is-active': id === selectedTab.id },
+                )}
+                data-cy="Tab"
+                key={id}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => id !== selectedTab.id && onTabSelected(tab)}
```

Comment:
I believe you should create a method for that

Reply:
done)